tests/bud.bats

```diff
@@ -519,6 +519,24 @@ _EOF
   expect_output --substring "Groups:	1000"
 }
 
+@test "build-test --mount=type=secret test relative to workdir mount" {
+  local contextdir=${TEST_SCRATCH_DIR}/bud/platform
+  mkdir -p $contextdir
+  echo "secret:foo" >$contextdir/secret1.txt
+  echo "secret:bar" >$contextdir/secret2.txt
+  cat > $contextdir/Dockerfile << _EOF
+FROM alpine
+RUN mkdir test
+WORKDIR test
+RUN --mount=type=secret,id=secret-foo,dst=secret1.txt --mount=type=secret,id=secret-bar,dst=secret2.txt \
+     cat /test/secret1.txt && cat /test/secret2.txt
```

Comment:
None of this content is changing, so they can just be regular files that live in the source tree instead of files we generate at test-time.